### ISSUE TITLE
Subtle grammar issue in 3.2.6

### ISSUE THREAD

**Issue by dtex** [w3c/wcag#2688]
> If a [web page](https://www.w3.org/TR/WCAG22/#dfn-web-page-s) contains any of the following help mechanisms, and those mechanisms are repeated on multiple web pages within a [set of web pages](https://www.w3.org/TR/WCAG22/#dfn-set-of-web-pages), they occur in the same relative order to other page content, unless a change is initiated by the user

As I understand it, this bit:

`they occur in the same relative order to other page content`

should be written

`they occur in the same order relative to other page content`

In the first example, "relative" modifies the word "order" which suggests the order is the same to a certain degree when compared with other things that have an order.

In the second, the phrase "relative to" more clearly establishes the relationship with "other page content" which is what was intended.

Happy to submit a PR if y'all agree this change is correct.

**Comment by cstrobbe** [comment]
The phrase [same relative order](https://w3c.github.io/wcag/guidelines/22/#dfn-same-relative-order) is in the glossary but there is no link to that glossary item. I'm not sure whether this is by accident or intentional.

The phrase "same relative order" is used in [SC 3.2.3 Consistent Navigation](https://w3c.github.io/wcag/guidelines/22/#consistent-navigation). In that SC, the order of those navigation mechanisms needs to be consistent among themselves. If you have navigation mechanisms A, B and C, which are repeated on several pages, they should always occur in the order A, B and C, i.e. in the same order relative to each other. You may insert navigation mechanism D in between so you end up with, e.g. A, B, D and C, but you can't put C before A or B.

SC 3.2.6 uses the phrase "the same relative order to other page content", so the criterion does not apply only to the order of the help mechanisms among themselves (if more than one is present). For this reason, the alternative wording "the same order relative to other page content" makes sense, but it wouldn't be bad if [Understanding SC 3.2.6](https://w3c.github.io/wcag/understanding/consistent-help.html) pointed out the difference in what "relative order" applies to in SC 3.2.6 and 3.2.3.

**Comment by bruce-usab** [comment]
I agree that either <q>same relative order</q> should link to the glossary term or that we should tweak 3.2.6 to not use that exact phrasing.

Substituting the term into SC, 3.2.6 becomes:

> If a web page contains any of the following help mechanisms ... they occur in the **same relative order** to other page content...

> If a web page contains any of the following help mechanisms ... they occur in the **same position relative to other items** to other page content, unless a change is initiated by the user...

That does not read correctly, so I think we should tweak 3.2.6.  Maybe:

> If a web page contains any of the following help mechanisms ... they occur in the **same <ins>position</ins> relative**  to other  page content...

**Comment by dtex** [comment]
Is there any value in my submitting a pull request for this? It's a small change, but I'm happy to do so if it is preferred.

**Comment by patrickhlauke** [comment]
i'd say it'd be good if you filed a PR (but understand that it may be a long time before it gets merged, as there's currently a huge backlog)

**Comment by bruce-usab** [comment]
Thanks @dtex for the PR!

Yes, there is currently a huge backlog.  There is a priority on Understanding for SC new with 2.2, so I expect this will get addressed before TR publication.  You spotting is very timely.

**Comment by mbgower** [comment]
> Personally, I am less comfortable with changing a defined term from 2.0 than I am with an editorial change to an SC new to 2.2.

Yes, I concur that the rewording of the current SC is the lighter change, both because as is stated clearly in this issue, it is grammatically more clear inverted -- one could make a good case this is 'just editorial', and because it does not impact the prior normative definition.
